ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -554,6 +569,7 @@ PrjFS_Result PrjFS_DeleteFile(
     
     if (nullptr == relativePath)
     {
+        LogWarning("PrjFS_DeleteFile: paramater relativePath should not be [NULL]");
```

Comment:
Anything that indicates a programmer error like this should, at a minimum, be logged as an error.